Gopkg.lock

```diff
@@ -7,11 +7,17 @@
   packages = ["quantile"]
   revision = "4c0e84591b9aa9e6dcfdf3e020114cd81f89d5f9"
 
+[[projects]]
+  name = "github.com/gogo/protobuf"
+  packages = ["proto"]
+  revision = "342cbe0a04158f6dcb03ca0079991a51a4248c02"
+  version = "v0.5"
+
```

Comment:
Is prometheus using both `gogo/protobuf` and `golang/protobuf` (below)?

Reply:
I am not so familiar to dep but this was added then I executed `dep ensure` and this file shall not be modified manually.

Reply:
Right. Ok, it may be from two different deps.